File: cmstestsuite/code/incorrect-fileio.cs
using System;
using System.IO;

public static class Batchfileio
{
    public static void Main()
    {
        StreamReader reader = new StreamReader("input.txt");
        int n = int.Parse(reader.ReadLine());
        StreamWriter writer = new StreamWriter("output.txt");
        writer.WriteLine("incorrect " + n);
        reader.Close();
        writer.Close();
    }
}

File: cmstestsuite/code/incorrect-fileio.java
import java.util.Scanner;
import java.io.PrintWriter;
import java.io.File;
import java.io.FileNotFoundException;

public class batchfileio {

    public static void main(String[] args) throws FileNotFoundException {
        File file = new File("input.txt");
        Scanner scanner = new Scanner(file);
        int n = scanner.nextInt();
        PrintWriter writer = new PrintWriter("output.txt");
        writer.println("incorrect " + n);
        writer.close();
    }
}
